### PR TITLE
Minor fixes for kubeadm reset

### PR DESCRIPTION
**Pull request by bart0sh**
This set of fixes was proposed by @neolit123 in [PR 63849](https://github.com/kubernetes/kubernetes/pull/63849) review
comments:

- Capitalized output after [reset]
- Used %v format for Go errors
- Fixed spelling: Trying using -> Trying to use

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
> Capitalized output after [reset]

so we do this in `init.go` and it seems better to capitalize some of these output sentences to the end user. same goes for the rest of the `stdout` info blobs, but not for the errors.

> Used %v format for Go errors

+1

> Fixed spelling: Trying using -> Trying to use

+1

**Comment by bart0sh** [comment]
@nebril > so we do this in init.go and it seems better to capitalize some of these output sentences to the end user. same goes for the rest of the stdout info blobs, but not for the errors.

done

**Comment by bart0sh** [comment]
@neolit123 > > so we do this in init.go and it seems better to capitalize some of these output sentences to the end user. same goes for the rest of the stdout info blobs, but not for the errors.

done

**Comment by neolit123** [comment]
@bart0sh 
looks good.

will wait on @fabriziopandini for approval.

thanks.

**Review by fabriziopandini — APPROVED** [review]
@bart0sh thanks for this PR!
/lgtm
/approve

**Pull request merged**
